pkgs/by-name/pi/pied/package.nix

```diff
@@ -25,11 +27,14 @@ flutter327.buildFlutterApplication rec {
     gst_all_1.gst-plugins-base
     gst_all_1.gst-plugins-good
     gst_all_1.gst-plugins-bad
+    killall
+    piper-tts
   ];
 
   postInstall = ''
     install -D flatpak/com.mikeasoft.pied.desktop -t $out/share/applications
     install -D flatpak/com.mikeasoft.pied.png -t $out/share/pixmaps
+    ln -s ${piper-tts}/bin/piper $out/bin/piper
```

Comment:
Shouldn't we use wrapProgram? Also for killall. Or does pied look next to it for piper?